hvplot/ui.py

```diff
@@ -688,10 +688,22 @@ def _plot(self):
         if len(df) > MAX_ROWS and not (
             self.kind in KINDS['stats'] or kwargs.get('rasterize') or kwargs.get('datashade')
         ):
-            df = df.sample(n=MAX_ROWS)
+            if self.kind == 'line':
+                param.main.param.warning(
```

Comment:
How about displaying these warnings in the alert box in addition/in place of as a programmatic warning?

Reply:
I suggest to do df = df.sample(n=MAX_ROWS).sort_index()`. Otherwise the line plot will not work as it should.

Reply:
I disagree with sort_index; the x may not always be index.

Reply:
good point. is there an option to add sort_index as an option? 
something like `df.hvplot.explorer(x='x', y='y', kind='line', sort_index=True)` 
where the defualt is `sort_index=False` ?

Reply:
I think it probably makes more sense to do `sort_values("x")` when sampling for line.